documentation/tools/dataprotector/methods/processprotecteddata.md

```diff
@@ -0,0 +1,108 @@
+# processProtectedData
```

Comment:
add a brief explanation of the return value of the function: "The task identifier, taskId, refers to a unique identifier associated with a task currently running on the iExec Bellecour blockchain. It uniquely identifies the specific execution of the application (data processing) on the blockchain."

Consider placing this explanation either here or in the ## Return value example section.